packages/utils/src/hex.ts

```diff
@@ -106,10 +97,10 @@ export function bytesToData(bytes: Uint8Array): DATA {
 /**
  * DATA as defined in ethereum execution layer JSON RPC https://eth.wiki/json-rpc/API
  */
-export function dataToBytes(hex: DATA, fixedLength: number | null): Uint8Array {
+export function dataToBytes(hex: DATA, fixedLength?: number): Uint8Array {
```

Comment:
keep the null type, forces you as consumer to think on the length. Else we'll forget to add it

Reply:
Reverted as you requested... 
but IMHO, I don't think it makes for a cleaner API having to pass null like this `dataToBytes(data.extraData, null)` as the `null` provides no information and might even be confusing when reading the code.